ParkingZoneApp/Models/Reservation.cs

```diff
@@ -29,5 +29,11 @@ public class Reservation
 
         [Required]
         public virtual ApplicationUser User { get; set; }
+
+        [NotMapped]
+        public bool IsActive 
+        { 
+            get => StartTime <= DateTime.Now && StartTime.AddHours(Duration) > DateTime.Now;
+        }
```

Comment:
why did you added it? where did you used it?

Reply:
it is for html i check whether is it active if it's so I'll make the background green as written in to do list